index.html

```diff
@@ -59,7 +59,7 @@
 
       <p>Most of the samples use <a href="//github.com/webrtc/adapter">adapter.js</a>, a shim to insulate apps from spec changes and prefix differences. (In fact, the standards and protocols used for WebRTC implementations are highly stable, and there are only a few prefixed names. For full interop information, see <a href="//www.webrtc.org/web-apis/interop">webrtc.org/web-apis/interop</a>.)</p>
 
-      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL will result in a PERMISSION_DENIED NavigatorUserMediaError.</p>
+      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL won't work.</p>
```

Comment:
Fails silently in Chrome and Opera.

Works in Firefox – don't think it used to. I've updated the text in #496.

On 14 April 2015 at 06:08, Justin Uberti <notifications@github.com> wrote:

> In index.html
> <https://github.com/webrtc/samples/pull/497#discussion_r28300978>:
>
> > @@ -59,7 +59,7 @@
> >
> >        <p>Most of the samples use <a href="//github.com/webrtc/adapter">adapter.js</a>, a shim to insulate apps from spec changes and prefix differences. (In fact, the standards and protocols used for WebRTC implementations are highly stable, and there are only a few prefixed names. For full interop information, see <a href="//www.webrtc.org/web-apis/interop">webrtc.org/web-apis/interop</a>.)</p>
> >
> > -      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL will result in a PERMISSION_DENIED NavigatorUserMediaError.</p>
> > +      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL won't work.</p>
>
> is an exception thrown, or does it fail silently?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/samples/pull/497/files#r28300978>.
>